CMakeLists.txt

```diff
@@ -133,23 +133,23 @@ option(INCLUDE_LINUX_I2C "Include I2C support" YES)
 IF (INCLUDE_LINUX_I2C)
 	CHECK_INCLUDE_FILES ("sys/types.h;linux/i2c-dev.h;linux/i2c.h" HAVE_LINUX_I2C_H)
 	IF (HAVE_LINUX_I2C_H)
-		message(STATUS "Building with I2C support")
+		message(SEND_ERROR "Building with I2C support")
 		add_definitions(-DHAVE_LINUX_I2C)
 	ELSE()
-		message(WARNING "I2C support disabled: headers not found!")
+		message(SEND_ERROR "I2C support disabled: headers not found!")
 	ENDIF (HAVE_LINUX_I2C_H)
 ENDIF (INCLUDE_LINUX_I2C)
 
-option(INCLUDE_SPI "Include SPI support" YES)
-IF (INCLUDE_SPI)
+option(INCLUDE_LINUX_SPI "Include SPI support" YES)
```

Comment:
I just made an error while copying the existing var `HAVE_LINUX_....` so that was/is already in the file. This update is just to keep variable names consistent. With your train of thought 'linux' should be removed from `HAVE_LINUX_...` as well.